src/main/java/com/duckblade/osrs/toa/util/OverlayUtil.java

```diff
@@ -0,0 +1,39 @@
+package com.duckblade.osrs.toa.util;
+
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Graphics2D;
+import net.runelite.api.Point;
+
+public final class OverlayUtil
+{
+	private OverlayUtil()
+	{
+	}
+
+	public static void renderTextLocation(final Graphics2D graphics2D, final Point point, final String text, final Color color, final int size, final int style, final boolean shadow)
```

Comment:
May as well give people a custom font option too if you're going to add all the other options. People will come bugging me about it otherwise

Reply:
I removed the font name so it will use a default